docs/command_reference.md

```diff
@@ -0,0 +1,235 @@
+<!--[metadata]>
++++
+title = "Notary Command Reference"
+description = "Notary command reference"
+keywords = ["docker, notary, command, notary command, reference"]
+[menu.main]
+parent="mn_notary"
+weight=99
++++
+<![end-metadata]-->
+
+# Notary Command Reference
+
+## Terminology Reference
+1. **GUN**: Notary uses Globally Unique Names (GUNs) to identify trusted collections.
+2. **Target**: Notary refers to the files the framework is used to distribute as "target files".  Target files are opaque to the framework. Whether target files are packages containing multiple files, single text files, or executable binaries is irrelevant to Notary.
+3. **Trusted Collection**: A trusted collection is a conceptual source of target files of interest to the application.
```

Comment:
Actually maybe I don't really know what a trusted collection means.  I interpreted it to be a collection of the targets in a notary repo, and maybe that's not what it actually means.  If it is, would it make sense to add `notary repository` here as a term and say how it's different than the trusted collection it represents?